### PR TITLE
Fix issue with `conda env` not displaying help (v2.1.2)

### LINKED ISSUE #56: conda env with no arguments
```
$conda env
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "/Users/aaronmeurer/anaconda/bin/conda-env", line 6, in <module>
    sys.exit(main())
  File "/Users/aaronmeurer/anaconda/lib/python3.4/site-packages/conda_env/cli/main.py", line 53, in main
    return args_func(args, parser)
  File "/Users/aaronmeurer/Documents/Continuum/conda/conda/cli/main.py", line 207, in args_func
    args.func(args, p)
AttributeError: 'Namespace' object has no attribute 'func'
```

### PR DESCRIPTION
**Pull request by tswicegood**
Thanks to @asmeurer for reporting.

Fixes #56.

### CODE REVIEW

**Comment by tswicegood** [comment]
*sighs* The build passes, just not on win32/64.  Merging and tagging.

**Comment by tswicegood** [comment]
Oops --- this should also merge to `master` since it's a bugfix, then into `develop`.

File: conda_env/cli/main.py [hunk 1]
```diff
@@ -35,6 +35,14 @@
 from . import main_update
 
 
+# TODO: This belongs in a helper library somewhere
```
Comment (asmeurer) [inline conda_env/cli/main.py]: Should args_func do this?

Comment (tswicegood) [inline conda_env/cli/main.py]: No -- you want to circumvent getting to the point of `args_func` by injecting the `--help` in to the actual args so it's there before its parsed.

Comment (tswicegood) [inline conda_env/cli/main.py]: That's not to say that you couldn't do it there, just if you want it to be handled automatically it should happen before the parse even happens.

**Pull request merged**